### ISSUE TITLE
Poly Loss

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#406]
**Short Description**
https://arxiv.org/abs/2204.12511

**Papers**
https://arxiv.org/abs/2204.12511

**Other Information**

still not clear if we want this or not, it looks promising from the abstract but we should probably do some sanity checking.

**Comment by bhack** [comment]
It seems good, we have already [3 Pytorch third party implementations](https://paperswithcode.com/paper/polyloss-a-polynomial-expansion-perspective-1).
But I don't know if we want to collect all the bleeding edge as fast as possible without a minimum of "sedimentation" (E.g. See the experience we had with the optimizers proliferation https://github.com/tensorflow/addons/issues/2203#issuecomment-711019703)

**Comment by LukeWood** [comment]
> It seems good, we have already [3 Pytorch third party implementations](https://paperswithcode.com/paper/polyloss-a-polynomial-expansion-perspective-1). But I don't know if we want to collect all the bleeding edge as fast as possible without a minimum of "sedimentation" (E.g. See the experience we had with the optimizers proliferation [tensorflow/addons#2203 (comment)](https://github.com/tensorflow/addons/issues/2203#issuecomment-711019703))

Yeah I agree.  This issue is a good tracking issue; but contribution definitely not yet welcome in that we should wait to see how things unfold.

**Comment by ianstenbit** [comment]
This issue is not currently tied to any near-term objectives of KerasCV, so we are closing it for now. If you think this was done in error, or you are interested in adding this feature yourself, please checkout our [README](https://github.com/keras-team/keras-cv/blob/master/README.md) and re-open this issue if it meets the listed criteria.

Thank you! - The KerasCV team
